universe/IDAllocator.cpp

```diff
@@ -305,7 +305,7 @@ void IDAllocator::SerializeForEmpire(Archive& ar, const unsigned int version, co
                 & BOOST_SERIALIZATION_NVP(m_offset_to_empire_id);
         } else {
             auto temp_empire_id = empire_id;
```

Comment:
is `temp_empire_id` needed?

Reply:
Yes, `empire_id` is a constant so it cann't be argument of `make_nvp` function.

Reply:
make it not const... it's a by value parameter anyway and isn't modified otherwise.

Reply:
Fixed